at_radial_gauges/README.md

```diff
@@ -0,0 +1,16 @@
+# radial_gauges
+
+A new Flutter project.
+
```

Comment:
Can we mention the types of Gauges offered and also talk about some of the important parameters?

Reply:
Thanks @sachins-geekyants , I've made the commit with the description of pubspec.yaml updated as requested.